code/game/jobs/job/civilians/support/doctor.dm

```diff
@@ -1,3 +1,9 @@
+//job options for doctors surgeon pharmacy technician(preparation of medecine and distribution)
+
+#define DOCTOR_VARIANT "Doc"
+#define SURGEON_VARIANT "Sur"
+//#define PHARMACY_TECHNICIAN_VARIANT "Pha"
```

Comment:
no commented out code

Reply:
a left over when i tried to make three possibility.